src/main/java/com/bulkbuy/enterprise/controller/NavigationController.java

```diff
@@ -39,19 +42,18 @@ public String getOrderLookup(Model model){
         return "orderLookup";
     }
 
-    @PostMapping(value="/saveOrder", consumes = "application/json", produces = "application/json")
-    public Order saveOrder(@RequestBody Order order) throws Exception {
+    @PostMapping(value="/saveOrder")
+    public String saveOrder(@ModelAttribute("saveOrder") Order savedOrder, Model model) {
         Order newOrder = null;
         try {
-            newOrder = orderService.create(order);
+            newOrder = orderService.create(savedOrder);
         }
         catch (Exception ex)
         {
             //TODO: LOGGING/Exception handling
             throw ex;
         }
-
-        return newOrder;
+        return "index";
```

Comment:
After the order has been posted, this will return the user to the index.html page.